desktop/apps/auction2/test/artwork.js

```diff
@@ -0,0 +1,26 @@
+require('babel-core/register')
+require('coffee-script/register')
```

Comment:
These two imports -- why here? Also, do these need to be imported at the `route` level, or can they sit more globally? (I was wondering about this this weekend, and was going to experiment... Also, I think if you're able to use `import` below this `require`, it means that `babel-core/register` is already imported somewhere else because this actually needs to sit one level up before the compiler will register ES6+. And one last tiny thing: you can keep use anonymous imports instead of require as they do the same thing:  

`import 'coffee-script/register'`